src/cli.js

```diff
@@ -147,13 +151,17 @@ export async function cmd (args, ac) {
     signal: ac?.signal
   }
 
+  let hasResult = false
   for await (const chunkResult of tcpExistsMany(endpoints.join(';'), options)) {
     for (const oneResult of chunkResult) {
-      process.stdout.write(formatOneResult(oneResult, delimiter, colorless))
+      if (!silent) {
+        process.stdout.write(formatOneResult(oneResult, delimiter, colorless))
+      }
+      hasResult = true
```

Comment:
For my use case, I only care about one port. But maybe others who specify multiple ports would want this to return true iff all listed ports are listening?